src/app/containers/Blocks/index.jsx

```diff
@@ -19,7 +20,7 @@ const Blocks = ({ blocks, componentsToRender }) =>
 
     return (
       <Block
-        key={id}
+        key={id || uuid()}
```

Comment:
this will generate a new key after every render, which isn't performance friendly and besides `id` is already a uuid because it's generated [here](https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/preprocessor/rules/addIdsToBlocks.js#L12), thing is the fixture data blocks used in storybook don't go through the pre-processor rules,  hence `id`'s don't get added, maybe modifying the fixture data to make use of uuids instead of adding it here